functional/FunctionalTest.java

```diff
@@ -137,72 +140,244 @@ public static String getRandomName() {
+    try {
+      String name = getRandomName();
+      client.makeBucket(name);
+      client.removeBucket(name);
+
+      mintLog("makeBucket(String bucketName)", 
+              null, 
+              System.currentTimeMillis() - startTime,
+              PASS,
+              null,
+              null,
+              null);
+
+    } catch (Exception e) {
+      mintLog("makeBucket(String bucketName)", 
+              null, 
+              System.currentTimeMillis() - startTime,
+              FAILED,
+              null,
+              null,
+              e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
+      throw e;
+    } 
   }
 
   /**
    * Test: makeBucket(String bucketName, String region).
    */
-  public static void makeBucket_test2() throws Exception {
-    System.out.println("Test: makeBucket(String bucketName, String region)");
+  public static void makeBucketwithRegion_test() throws Exception {
+
+    if (!mintEnv) {
+      System.out.println("Test: makeBucket(String bucketName, String region)");
+    }
+    long startTime = System.currentTimeMillis();
 
-    String name = getRandomName();
-    client.makeBucket(name, "eu-west-1");
-    client.removeBucket(name);
+    try {
+      String name = getRandomName();
+      client.makeBucket(name, "eu-west-1");
+      client.removeBucket(name);
+
+      mintLog("makeBucket(String bucketName, String region)", 
+              "region: eu-west-1", 
+              System.currentTimeMillis() - startTime,
+              PASS,
+              null,
+              null,
+              null);
+    } catch (Exception e) {
+      mintLog("makeBucket(String bucketName, String region)", 
+              "region: eu-west-1", 
+              System.currentTimeMillis() - startTime,
+              FAILED,
+              null,
+              null,
+              e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
+      throw e;
+    }    
   }
 
   /**
    * Test: makeBucket(String bucketName, String region) where bucketName has
    * periods in its name.
    */
-  public static void makeBucket_test3() throws Exception {
-    System.out.println("Test: makeBucket(String bucketName, String region)");
+  public static void makeBucketWithPeriod_test() throws Exception {
+
+    if (!mintEnv) {
+      System.out.println("Test: makeBucket(String bucketName, String region)");
+    }
 
+    long startTime = System.currentTimeMillis();
     String name = getRandomName() + ".withperiod";
-    client.makeBucket(name, "eu-central-1");
-    client.removeBucket(name);
+
+    try {
+      client.makeBucket(name, "eu-central-1");
+      client.removeBucket(name);
+
+      mintLog("makeBucket(String bucketName, String region)", 
+                          "name: " + name + ", region: eu-central-1", 
+                          System.currentTimeMillis() - startTime,
+                          PASS,
+                          null,
+                          null,
+                          null);
```

Comment:
Passing args in new line seems a little weird to me. Any specific reasons for doing this? If not we can follow the style we follow for other API calls - `client.makeBucket(name, "eu-central-1");`